### DISCUSSION TITLE
CTD after pressing print wx report

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5872] [General]
Hi dev team,

since now I was glad after all updates of microsoft in the near past that I don´t have any CTD´s ... Today while pressing print button for the wx report I got my first CTD :-( .... Interesting that I pressed it during other session and could continue the flight without CTD ... I am using the followig dev version:

![grafik](https://user-images.githubusercontent.com/76531893/134158663-30b7c770-d7ac-48d5-adc7-021683dee765.png)

greets

**Comment by Dale-mo** [comment]
Same problem here.  I have reverted to v7 stable for now
Dale

**Comment by MapleNAV-TCASFAULT** [comment]
https://github.com/flybywiresim/a32nx/issues/5429 you can check this out to find maybe a bit more info, but this is already a known issue that occurs to some people.
